modules/configuration/pages/error_handling.adoc

```diff
@@ -1,75 +1,94 @@
+You can use the xref:components:processors/try.adoc[`try` processor] to define a list of processors that are executed in sequence. If a processor fails for a particular message, that message skips the remaining processors.
+
+For example:
+
+- If `processor_1` fails to process a message, that message skips `processor_2` and `processor_3`.
+- If a message is processed by `processor_1`, but `processor_2` fails, that message skips `processor_3`, and so on. 
 
 [source,yaml]
 ----
 pipeline:
   processors:
     - try:
-      - resource: foo
-      - resource: bar # Skipped if foo failed
-      - resource: baz # Skipped if foo or bar failed
+      - resource: processor_1
+      - resource: processor_2 # Skip if processor_1 fails
+      - resource: processor_3 # Skip if processor_1 or processor_2 fails
 ----
 
 == Recover failed messages
 
-Failed messages can be fed into their own processor steps with a xref:components:processors/catch.adoc[`catch` processor]:
+You can also route failed messages through defined processing steps using a xref:components:processors/catch.adoc[`catch` processor].
+
+For example, if `processor_1` fails to process a message, it is rerouted to `processor_2`.
 
 [source,yaml]
 ----
 pipeline:
   processors:
-    - resource: foo # Processor that might fail
+    - resource: processor_1 # Processor that might fail
     - catch:
-      - resource: bar # Recover here
+      - resource: processor_2 # Processes rerouted messages
 ----
 
-Once messages finish the catch block they will have their failure flags removed and are treated like regular messages. If this behavior is not desired then it is possible to simulate a catch block with a xref:components:processors/switch.adoc[`switch` processor]:
+After messages complete all processing steps defined in the `catch` block, failure flags are removed and they are treated like regular messages.
+
+To keep failure flags in messages, you can simulate a `catch` block using a xref:components:processors/switch.adoc[`switch` processor]:
 
 [source,yaml]
 ----
 pipeline:
   processors:
-    - resource: foo # Processor that might fail
+    - resource: processor_1 # Processor that might fail
     - switch:
       - check: errored()
         processors:
-          - resource: bar # Recover here
+          - resource: processor_2 # Processes rerouted messages
 ----
 
 == Logging errors
 
-When an error occurs there will occasionally be useful information stored within the error flag that can be exposed with the interpolation function xref:configuration:interpolation.adoc#bloblang-queries[`error`]. This allows you to expose the information with processors.
+When an error occurs, there may be useful information stored in the error flag. You can use xref:guides:bloblang/functions.adoc#error[`error`] Bloblang function interpolations to write this information to logs. You can also add the following Bloblang functions to expose additional details about the processor.
```

Comment:
> You can also add the following Bloblang functions to expose additional details about the processor.

You can also use the following Bloblang functions to expose additional details about the processor which triggered the error: